packages/theme/src/cli/commands/theme/profile.ts

```diff
@@ -0,0 +1,75 @@
+import {themeFlags} from '../../flags.js'
+import ThemeCommand from '../../utilities/theme-command.js'
+import {profile} from '../../services/profile.js'
+import {ensureThemeStore} from '../../utilities/theme-store.js'
+import {findOrSelectTheme} from '../../utilities/theme-selector.js'
+import {renderTasksToStdErr} from '../../utilities/theme-ui.js'
+import {ensureAuthenticatedThemes} from '@shopify/cli-kit/node/session'
+import {Flags} from '@oclif/core'
+import {globalFlags, jsonFlag} from '@shopify/cli-kit/node/cli'
+import {Task} from '@shopify/cli-kit/node/ui'
+
+export default class Profile extends ThemeCommand {
+  static summary = 'Profile the Liquid rendering of a theme page.'
+
+  static usage = ['theme profile --url /products/classic-leather-jacket']
```

Comment:
The url is not mandatory, so I think we should add the usage without flags. That way, I think [this README](https://github.com/Shopify/cli/pull/5109/files#diff-15986190ef9581ab59bcd5483b2c09e7fd0bd439d6f6cddbc94b0b1de094ee51R76) won't include the flag in the command title, which is weird.

```suggestion
  static usage = ['theme profile', 'theme profile --url /products/classic-leather-jacket']
```